network db rewrite (#12534)
containers/podman: docs/source/markdown/podman-create.1.md

Specify a static IP address for the container, for example **10.88.64.128**.
This option can only be used if the container is joined to only a single network - i.e., `--network=_network-name_` is used at most once -
and if the container is not joining another container's network namespace via `--network=container:_id_`.
The address must be within the network's IP address pool (default **10.88.0.0/16**).

> To use more than one network together with a static ip address use the *ip* option described under **--network**.

Reviewer: I think you're saying:
```suggestion
To specify multiple static IP addresses per container, set multiple networks using the `**--network**` option with a static IP address specified for each using the `IP` mode for that option.

```

#### **--mac-address**=*address*

Container network interface MAC address (e.g. 92:d0:c6:0a:29:33)
This option can only be used if the container is joined to only a single network - i.e., `--network=_network-name_` is used at most once -
and if the container is not joining another container's network namespace via `--network=container:_id_`.

Remember that the MAC address in an Ethernet network must be unique.
The IPv6 link-local address will be based on the device's MAC address
according to RFC4862.

> To use more than one network together with a static mac address use the *mac* option described under **--network**.

Reviewer: If you take the IP suggestion above, do similar here.

Valid _mode_ values are:

> - **bridge[:OPTIONS,...]**: Create a network stack on the default bridge. This is the default for rootfull containers. It is possible to specify these additional options:

Reviewer: It would be good to have an example of one of these below, especially the `IP` or the `mac` as we talk about them earlier.

  - **alias=name**: Add network-scoped alias for the container.
  - **ip=IPv4**: Specify a static ipv4 address for this container.
  - **ip=IPv6**: Specify a static ipv6 address for this container.
  - **mac=MAC**: Specify a static mac address address for this container.
  - **interface_name**: Specify a name for the created network interface inside the container.

> - **\<network name or ID\>[:OPTIONS,...]**: Connect to a user-defined network, this is just the network name or ID from a network created by **[podman network create](podman-network-create.1.md)**. Using the network name implies the network mode bridge. It is possible to specify same options as described under bridge above. You can use **--network** multiple times to specify additional networks.

Reviewer: - **\<network name or ID\>[:OPTIONS,...]**: Connect to a user-defined network; this is the network name or ID from a network created by **[podman network create](podman-network-create.1.md)**. Using the network name implies the bridge network mode. It is possible to specify the same options described under the bridge mode above. You can use the **--network** option multiple times to specify additional networks.

#### **--network-alias**=*alias*

> Add network-scoped alias for the container. This will set the alias for all networks that the container joins. To set a name only for a specific network use the alias option as described under **--network**.

Reviewer: ```suggestion
Add a network-scoped alias for the container, setting the alias for all networks that the container joins. To set a name only for a specific network, use the alias option as described under the **--network** option.
```

> Network aliases work only with the bridge networking mode. This options can be specified multiple times.

Reviewer: ```suggestion
Network aliases work only with the bridge networking mode. This option can be specified multiple times.
```
